### DISCUSSION TITLE
School Chromebooks auto-update, No method works.

### DISCUSSION THREAD

**Discussion by Wynfhi** [3kh0/ext-remover#59] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Title, I've tried every method. Even other random ones found, none work. Checked policys and js is not blocked, I have used Js to edit the page on webstorex and that worked whilst the deleter doesn't.

**Comment by 3kh0** [comment, marked as answer]
What chromebook version are you on, it might have been patched.

**Reply by Wynfhi** [reply]
106.0.5249.112

**Reply by 3kh0** [reply]
It might have been patched

**Reply by Sikkox** [reply]
Is there anyway we can revert back to 105? My friend didn't update his chromebook and it was on Version 105 and It both worked for them.

**Reply by 3kh0** [reply]
Go to https://chrome.dev

**Reply by capalothussein** [reply]
https://chrome.dev/ doesn't work

**Reply by 3kh0** [reply]
@capalothussein what do you mean it does not work?

**Reply by ghost** [reply]
chrome.dev doesnt load for me

**Reply by 3kh0** [reply]
@ttvbeghhf use it on your home computer, or use a proxy

**Reply by ghost** [reply]
what do you mean

**Reply by chardynamics** [reply]
What part of that do you not understand
